runtime/include/blueoil.hpp

```diff
@@ -23,9 +23,17 @@ extern "C" {
 
 
 namespace blueoil {
-struct Tensor {
+class Tensor {
+public:
   std::vector<float> data;
   std::vector<int> shape;
+  int elems();
+  static Tensor zeros(std::vector<int> shape);
```

Comment:
why it is not a constructor?

Reply:
because imitate numpy API. But I changed it to constructor.
https://github.com/blue-oil/blueoil/pull/9/commits/8f5da4e01b17a27549d1e55ecd4faedc259a7b91